gke-nfs-lb/charts/README.md

```diff
@@ -0,0 +1,15 @@
+# Install NFS CSI LB driver with Helm
+
+## Prerequisites
+ - [install Helm](https://helm.sh/docs/intro/quickstart/#install-helm)
+
+### install a specific version
```

Comment:
Add TODO for upgrades and/or updates to the IP list

Reply:
done